Makefile

```diff
@@ -19,6 +19,12 @@ SOURCES ?= gs://wolfi-sources/
 ifneq (${MELANGE_RUNNER},)
 	MELANGE_OPTS += --runner ${MELANGE_RUNNER}
 endif
+QEMU_KERNEL_IMAGE ?= kernel/boot/vmlinuz
+ifeq (${MELANGE_RUNNER},qemu)
+	QEMU_KERNEL_DEP = ${QEMU_KERNEL_IMAGE}
+	export QEMU_KERNEL_IMAGE
+endif
```

Comment:
Does this magic make it so that we fetch the kernel when the melange runner is set to QEMU?  Amazing 🤩